nx-X11/programs/Xserver/dix/dispatch.c

```diff
@@ -1370,6 +1379,7 @@ ProcQueryFont(register ClientPtr client)
 	    return(BadAlloc);
 	}
 
+	memset(reply, 0, rlength);
```

Comment:
No. xcalloc() is defined in include/os.h as

#define xcalloc(_num, _size) Xcalloc((unsigned long)(_num)*(unsigned long)(_size))

So xalloc(1, 5) will alloc 1*5 Bytes and clear them with zeros.

memset(..,1,5) however would fill the memory with 0x01 0x01 0x01 0x01 0x01